Source/WebKitLegacy/mac/WebView/WebResource.mm

```diff
@@ -338,12 +338,12 @@ - (NSString *)_suggestedFilename
 #if !PLATFORM(IOS_FAMILY)
 - (NSFileWrapper *)_fileWrapperRepresentation
 {
-    auto wrapper = adoptNS([[NSFileWrapper alloc] initRegularFileWithContents:[self data]]);
+    NSFileWrapper *wrapper = [[NSFileWrapper alloc] initRegularFileWithContents:[self data]];
     NSString *filename = [self _suggestedFilename];
     if (!filename || ![filename length])
         filename = [[self URL] _webkit_suggestedFilenameWithMIMEType:[self MIMEType]];
     [wrapper setPreferredFilename:filename];
-    return wrapper.autorelease();
+    return adoptNS(wrapper).autorelease();
```

Comment:
I disagree with some of the changes in this patch. We should adopt as early as possible.

Reply:
Okay then. Reverting those changes.